### DISCUSSION TITLE
Sonar distortion and false readings while psychotic

### DISCUSSION THREAD

**Discussion by Veritius** [FakeFishGames/Barotrauma#4695] [Ideas and suggestions]
Simple as the title suggests, really. Sonar isn't accurate or even completely wrong while the user is psychotic.

**Comment by Saevon** [comment]
Mess with status readouts as well!  fake flooding / hull damage / air quality.

Perhaps it shows water flooding randomly too!  Ducts, platforms, walls, shells would all have a chance to show the "damaged & flooding" animation and spawn water just for the current player.

**Comment by NilanthAnimosus** [comment]
I think instead, having fake-entities (Fake monsters basically) in the ocean would be a nice psychosis addition. They can appear on sonar and would mimic the same effect. But also does not actually impact the game (IE. the submarine just goes through it, like on collission or reaching the player it fades away) - would be fun watching psychotic captains ram molochs to find out the next one is, in fact. real.

These could act just like regular monster spawns, just with vastly simplified AI and clientside in nature (Or server side markers so other psychotics see the same thing, but not sure there would be a good reason for that)

I always have a problem with psychosis on that it generally is not believable, rooms randomly flood without damage, fires appear with no effect or real reason associated. You can instantly work out whats fake and continue like you don't even have it. Monsters in the water? Hard to ignore that one!
